src/models.ts

```diff
@@ -55,13 +41,11 @@ export type MutationHook = (
 export type ObjectModel = BaseModel & {
   type: 'object';
   interfaces?: string[];
-  // createdAt, createdBy, updatedAt, updatedBy can be null
-  nonStrict?: boolean;
   queriable?: boolean;
   listQueriable?: boolean;
-  creatable?: boolean;
-  updatable?: boolean;
-  deletable?: boolean;
+  creatable?: boolean | { createdBy?: Partial<ModelField>; createdAt?: Partial<ModelField> };
```

Comment:
Not sure about the naming: 
```suggestion
  creatable?: boolean | { byRole?: Partial<ModelField>; createdAt?: Partial<ModelField> };
```

Reply:
Ah, one moment, this is not the role, this is the actual user id, right?